### ISSUE TITLE
Gitlab helm chart upgrade breaks things

### ISSUE THREAD

**Issue by mattmattox** [rancher/rancher#23217]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- upgrade the gitlab helm chart to the latest version (2.3.2) and runner to the latest version (0.9.0)

**Result:**
Wait helm template failed. Error: parse error in "gitlab/charts/gitlab-runner/templates/_helpers.tpl": template: gitlab/charts/gitlab-runner/templates/_helpers.tpl:63: unexpected "=" in operand : exit status 1

**Other details that may be helpful:**
https://gitlab.com/gitlab-org/charts/gitlab-runner/issues/88

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.2.8
- Installation option (single install/HA): HA

**Comment by deniseschannon** [comment]
This issue seems to be that using the latest version of Helm wouldnt allow this chart to be deployed correctly.

As of v2.3.2, we've upgraded Helm to a newer version, so this should be fixed.

**Comment by jiaqiluo** [comment]
The bug is reproduced in v2.2.8

- run Rancher v2.2.8 single install
- add an RKE cluster 
- add the Gitlab helm repo:  `gitlab    https://charts.gitlab.io/    master`
- depoly the following apps:
```
- gitlab v2.3.2
- gitlab-runner v0.9.0
```

results:
- both apps cannot be deployed successfully with the same errors as the issue provides
<img width="1400" alt="Screen Shot 2019-11-11 at 11 46 50 AM" src="https://user-images.githubusercontent.com/6218999/68612295-7349ed00-0479-11ea-8acf-37dae387cf32.png">
<img width="1395" alt="Screen Shot 2019-11-11 at 11 47 30 AM" src="https://user-images.githubusercontent.com/6218999/68612296-7349ed00-0479-11ea-87fa-0cbf0873ad66.png">

**Comment by jiaqiluo** [comment]
The bug fix is validated in both v2.3.2 and v2.3-head 5187fce31

Follow the same steps, the mentioned error does not show up anymore.  

also tested with the latest version for each app and not see the mentioned error:
```
- gitlab v2.4.6
- gitlab-runner v0.10.0
```

**Comment by mattmattox** [comment]
I have confirmed that upgrading to 2.3.2 and resolved the issue.
